src/components/Nav/NavList/NavList.js

```diff
@@ -116,6 +116,15 @@ export default class NavList extends Component {
         role="menuitem"
       >
         <div className={`${navItemNamespace}__link`}>
+          <div>
+            {this.props.hasIcon && (
+              <img
+                alt={'section1'}
+                className={`${navItemNamespace}__img`}
+                src={this.props.icon}
```

Comment:
you should deconstruct your props so that your code looks cleaner. You can see this being done on line 99 already